### PR TITLE
ENH: gracefully recover from trying to use non-existent GPU.

### PR DESCRIPTION
**Pull request by Leengit**
ENH: Support initializers: empty, zeros, ones, rand, randn
STYLE: Removing definition and use of TorchVoid_t implementation of void_t
ENH: Build pytorch tensors for bool

### CODE REVIEW

**Comment by Leengit** [comment]
This builds and tests on my local system.  Github failures are known bugs independent of this change.

File: wrapping/itkTorchImage.wrap [hunk 1]
```diff
@@ -1,4 +1,4 @@
-set(WRAP_ITK_TORCH_SCALAR "F" "D" "UC" "SC" "SS" "SL" "SLL")
+set(WRAP_ITK_TORCH_SCALAR "F" "D" "B" "UC" "SC" "SS" "SL" "SLL")
```
Comment (dzenanz) [inline wrapping/itkTorchImage.wrap:1]: Is `B` supported and interchangeable with `UC`?

**Review by dzenanz — APPROVED** [review]

File: wrapping/itkTorchImage.wrap (hunk 1 above)
Comment (Leengit) [reply wrapping/itkTorchImage.wrap:1]: [This documentation page](https://pytorch.org/docs/stable/tensors.html) indicates that `torch.bool` is distinct from `torch.uint8` and from `torch.int8`, at least at the interface level.  I don't know how Torch handles the underlying implementation.

Comment (dzenanz) [reply wrapping/itkTorchImage.wrap:1]: I thought that B is for `byte`, not `bool` 😄

**Pull request merged**